### ISSUE TITLE
twisted/test/stdio_test_halfclose.py typo

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11248]
|<img alt="rikyu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| rikyu reported|
|-|-|
|Trac ID|trac#1740|
|Type|defect|
|Created|2006-05-22 21:31:23Z|

So, I had brought this up on IRC awhile back, but it appears that
    source:/trunk/twisted/test/stdio_test_halfclose.py@current#L20

has the string "???" inserted as a placeholder.

This is especially an issue for Debian, as every time you install a Python-related package it recompiles the site-packages directory, and if that fails, it tends to stop the package install/config process.

<details><summary>Searchable metadata</summary>

```
trac-id__1740 1740
type__defect defect
reporter__rikyu rikyu
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1148333483000000 1148333483000000
changetime__1148334447000000 1148334447000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [16921]) remove file which does not do anything

Author: exarkun
Reviewer: radix
Fixes [#11248](https://github.com/twisted/twisted/issues/11248)
Refs [#1742](https://github.com/twisted/twisted/issues/1742)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
